specs/phase1/beacon-chain.md

```diff
@@ -671,7 +671,7 @@ def is_winning_attestation(state: BeaconState,
     ``winning_root`` formed by ``committee_index`` committee at the current slot.
     """
     return (
-        attestation.data.slot == state.slot
+        is_on_time_attestation(state, attestation)
```

Comment:
Hmmm, just found here is a typing error, the given attestation is `PendingAttestation`, but `is_on_time_attestation` takes `Attestation`. We can either:
1. change this line to `attestation.data.slot == compute_previous_slot(state.slot)` or
2. make `is_on_time_attestation` take `attestation: AttestationData`

Reply:
@terencechain any thought? :)

Reply:
I think 2 is nicer. This reduces disparity and avoids multiple ways of checking whether attestations (pending or not) is on time or not 👍